test/Riok.Mapperly.Tests/Mapping/IgnoreObsoleteTest.cs

```diff
@@ -345,6 +345,7 @@ class B
             );
     }
 
+#if ROSLYN4_4_OR_GREATER
```

Comment:
IsRequired is polyfilled for `Roslyn3.9`, why doesn't this test work? Does the polyfill work correctly?